packages/features/form-builder/FormBuilderField.tsx

```diff
@@ -311,6 +311,10 @@ export const ComponentForField = ({
     if (!field.optionsInputs) {
       throw new Error("Field optionsInputs is not defined");
     }
+    const options = field.options.map((field) => {
+      return { ...field, value: field.value === "inPerson" ? field.label : field.value };
```

Comment:
Could you check if the organizer address value is correctly shown after creating booking. Also, in email.

I think with just this change, handleNewBooking wouldn't know which location was selected as it looks up by type

Reply:
yeah i checked. i got the correct location

<img width="648" alt="Screenshot 2023-09-12 at 5 49 05 PM" src="https://github.com/calcom/cal.com/assets/53316345/d989762d-6d40-4666-9c8b-8100eac88cc9">

Reply:
<img width="648" alt="Screenshot 2023-09-12 at 5 49 59 PM" src="https://github.com/calcom/cal.com/assets/53316345/5a0fbcce-d4c8-4f22-9402-968a203bc36c">

Reply:
cc: @hariombalhara

Reply:
So, the problem is that `handleNewBooking` still doesn't have a clue which type of location was selected. It just knows that some text was given. That text can be an organizer address or organizer phone number or some other type of location. In case it can't detect which location type it is it just shows what was given to it.

So, even though it works, we don't logically know if it's an address, phone, or something else that was selected.

Also one thing to note is that there would be a change in the response that the webhooks receives for responses. Earlier it would have been **inPerson**, now it would be the actual value say 'New York'. I am not sure if people would be relying on it.

It seems to be good as a hotfix, but we need to fix this properly.

I am trying a few things before approving.

Reply:
I pushed a change to send _type along. Could you review that

Reply:
Leave it, I reverted my change. I think easiest would be for us to use just the slugified label as ID for location and it's practical to assume that locations would have different labels. If needed we can work on having an ID later.

I think an important followup would be to use label only for all types of locations and change everywhere to do lookup based on label instead of type of location.